src/docs/testing.mdx

```diff
@@ -37,6 +43,31 @@ the APIs our customers, integrations and front-end application continue to work
 endeavour to include tests that cover the various user roles, and cross organization/team access scenarios,
 as well as invalid data scenarios as those are often overlooked when manually testing.
 
+### Running pytest
+
+You can use pytest to run a single directory, single file, or single test depending on the scope of your
+changes:
+
+```shell
+# Run tests for an entire directory
+pytest tests/sentry/api/endpoints/
+
+# Run tests for all files matching a pattern in a directory
+pytest tests/sentry/api/endpoints/test_organization_*.py
+
+# Run test from a single file
+pytest tests/sentry/api/endpoints/test_organization_group_index.py
+
+# Run a single test
+pytest tests/sentry/api/endpoints/test_organization_group_index.py
+pytest tests/snuba/api/endpoints/test_organization_events_distribution.py::OrganizationEventsDistributionEndpointTest::test_this_thing
+
+# Run all tests in a file that match a substring
+pytest tests/snuba/api/endpoints/test_organization_events_distribution.py -k method_name
+```
+
+Refer to the [pytest](http://doc.pytest.org/en/latest/usage.html) docs for more usage options.
```

Comment:
`-s` is pretty useful and not obvious, if you don't wanna add this, I'll make a note to self to add in future.

Reply:
I can add notes for -s and -k as those are two I find myself using all the time.